scripts/datadotparl/update-members

```diff
@@ -71,6 +71,13 @@ class Position(object):
     self.position = position
     self.list = positions
 
+  def chair_check(self, chair_date, periods, to_date):
+    if chair_date[1] < to_date:
+      periods.append({'position': 'Chair', 'fromdate': chair_date[0], 'todate': chair_date[1]})
+      periods.append({'position': '', 'fromdate': chair_date[1], 'todate': to_date})
+    else:
+      periods.append({'position': 'Chair', 'fromdate': chair_date[0], 'todate': to_date})
+
```

Comment:
The only entry with two chair dates at present is: Member for a week, Chair for a year, Member for a few months (yes, this presumably was over election period, but doesn't have to be), Chair until now. The code should handle this correctly with the membership in-between the two chairs?